test_library.py

```diff
@@ -0,0 +1,111 @@
+import unittest
+
+from Library import *
+
+class TestStringMethods(unittest.TestCase):
+  def setUp(self):
+    self.library = Library([],[],[])
+
+  def test_new_library(self):
+    self.assertEqual(len(self.library.series), 0)
+    self.assertEqual(len(self.library.series), 0)
+    self.assertEqual(len(self.library.songs), 0)
+
+  def test_add_remove(self):
```

Comment:
it might be better to test add and remove separately

Reply:
They are tested separately in other tests, this is an integration test. I've moved it to its separated category for clarity